alibi/explainers/cfproto.py

```diff
@@ -758,13 +756,13 @@ def fit(self,
 
                 # multidim scaled distances
                 d_abs_abdm, _ = multidim_scaling(d_abdm, n_components=2, use_metric=True,
-                                                 feature_range=self.feature_range,
+                                                 feature_range=self.feature_range,  # type: ignore
```

Comment:
What was the error here? I thought `multidim_scaling` takes tuples?

If can't be resolved then would be great to narrow down the error code (i.e. `type: ignore[error-code] (same for below).

Reply:
`Argument "feature_range" to "multidim_scaling" has incompatible type "Tuple[Any, ...]"; expected "Tuple[Any, Any]"`
Due to for loop being used.

Reply:
OK, let's add the correct `error-code` to the new type ignore's then.